tjdashboard/code/site/views/dashboard/view.html.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @version    CVS: 1.0.0
+ * @package    Com_Tjdashboard
+ * @author     Techjoomla <extensions@techjoomla.com>
+ * @copyright  2017 Techjoomla
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+// No direct access
+defined('_JEXEC') or die;
+
+jimport('joomla.application.component.view');
+
+/**
+ * View to edit
+ *
+ * @since  1.6
+ */
+class TjdashboardViewDashboard extends JViewLegacy
+{
+	protected $state;
+
+	protected $item;
+
+	protected $form;
+
+	/**
+	 * Display the view
+	 *
+	 * @param   string  $tpl  Template name
+	 *
+	 * @return void
+	 *
+	 * @throws Exception
+	 */
+	public function display($tpl = null)
+	{
+		$app = JFactory::getApplication();
+		$currentMenuItem = $app->getMenu()->getActive();
+		$params = $currentMenuItem->params;
+		$this->dashboardId = $params->get('dashboard_id', 0, 'INT');
```

Comment:
If I don't create a link to a specific dashboard, the id will be in the URL, how is that handled ?

Reply:
@coolbung Yes agree wanted to see the demo of it so I could able to solve it , as well as the permission related things also wanted to see it here in html.